### ISSUE TITLE
Addon not showing up in UI after repository added.

### ISSUE THREAD

**Issue by a1g0r1thm9** [dbuezas/ha-addons#8]
![image](https://github.com/dbuezas/ha-addons/assets/148662538/6d61cfb4-a237-4aad-9a8b-0fb0d7991b10)
I set up my DNS, but after adding the github repo to my addons the cam addon doesnt appear under to repo name.

**Comment by dbuezas** [comment]
That's very odd, are you sure you entered the right repo url?

**Comment by a1g0r1thm9** [comment]
Yes, I added the url included in the readme. It also finds the repo, but doesn't seem to find the addon. I'm using HA supervised on debian 12.

**Comment by a1g0r1thm9** [comment]
I figured it out! I had to clone this repo and comment out the "advanced" option in the config.yaml of the addon. I'm sure there's an easier way, but this works for me (for now).

**Comment by dbuezas** [comment]
Oh, you don't have advanced mode enabled then.
You may want to try it out: 
In the left bar, click on your name and then enable it. You'll get a lot more options in the home assistant UI :)
